cabal.project

```diff
@@ -84,7 +84,7 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-node
-  tag: 4f44b01f190030dd56f40edb462a2addb6df4061
+  tag: a819311473563cb2ab3cd91543cf0f63facdf43e
```

Comment:
That is not going to happen. You are downgrading from a GH from Oct 13th to one from Oct 5th.

One of the next things I need to do is mode to an even more recent recent version of `node` in order to support Allega/Mary.

Reply:
So you are saying your releases don't correspond to any of the snapshots cardano-node and cardano-wallet are using? Can you explain to me how I can include cardano-db-sync in cardano-wallet when all dependencies are different?

Reply:
`4f44b01f190030dd56f40edb462a2addb6df4061` doesn't seem to be any released cardano-node version (correct me if I missed something) and @dcoutts explained on slack that we should only use released versions.

Reply:
> So you are saying your releases don't correspond to the snapshots cardano-node and cardano-wallet are using?

Until this week, building with `cardano-wallet` was never even the on the radar.

> Would you be able to explain to me how I can include cardano-db-sync in cardano-wallet

I would strongly advise you not to. Neither `wallet` nor `db-sync` ever considered this before this week.

> 4f44b01f190030dd56f40edb462a2addb6df4061 doesn't seem to be any released cardano-node version (correct me if I missed something) and @dcoutts explained on slack that we should only use released versions.

The `master` branch of `db-sync` will occassionally **need** to depend on `node` git hashes that are not formal releases. You should avoid depending on those. Similarly you should avoid depending on non-release versions of `db-sync`.

So what **release** of `db-sync` are you using?